app/pods/publications/index/template.hbs

```diff
@@ -25,6 +25,17 @@
     </Auk::Toolbar::Group>
   </Auk::Toolbar>
 </Auk::Navbar>
+{{#if this.publicationsFilter.isFiltered}}
+  <Auk::Navbar class="auk-publication-overview-navbar" @skin="gray-100" @auto={{true}}>
```

Comment:
waarom  `auk-publication-overview-navbar`
Deze geeft een `background-color: transparent`
maar `skin=gray-100` geeft `background: $au-gray-100`

Die css klasse wordt weer enkel hier gebruikt, en die klasse is 'to be removed'

Reply:
ik ben er ook van overtuigd dat die css klasse niet nodig is op die andere plaats, want daar zit nergens een auk-menu (Auk-dropdown) onder.  Maar is buiten scope :)